src/platforms/react-native/upload-debug.mdx

```diff
@@ -10,12 +10,12 @@ Debug symbols help provide you with readable stack traces, which Sentry displays
 
 With default settings, complete stack traces are available in your Java/Kotlin error, out of the box, unless you use [enableProguardInReleaseBuilds](https://reactnative.dev/docs/signed-apk-android#enabling-proguard-to-reduce-the-size-of-the-apk-optional). In this case, you need to upload the ProGuard/R8 mapping files generated by the [Android Gradle Plugin](https://developer.android.com/studio/build/shrink-code#shrink-code), so Sentry can show you proper stack traces.
 
-Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, dwarf debug files need to be uploaded for an Android app with [NDK](https://developer.android.com/ndk/) support.
-
 ## Uploading With Sentry Gradle Plugin
 
 Upload the [Android ProGuard/R8 mapping files and native symbols](/platforms/android/proguard/) by the recommended method of using our Gradle integration.
 
+Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, dwarf debug files need to be uploaded for an Android app with [NDK](https://developer.android.com/ndk/) support. Enabled native symbols upload for React Native Android builds [using Sentry Gradle Plugin `uploadNativeSymbols` option](/platforms/react-native/manual-setup/manual-setup/#enable-sentry-agp). To enable [Source Context](/platforms/apple/guides/android/data-management/debug-files/source-context/) use `includeSourceContext`.
```

Comment:
```suggestion
Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, DWARF debug files need to be uploaded for an Android app with [NDK](https://developer.android.com/ndk/) support.

To upload native symbols for React Native Android builds, use the Sentry Gradle Plugin's [`uploadNativeSymbols` option](/platforms/react-native/manual-setup/manual-setup/#enable-sentry-agp).

To enable Source Context use the [`includeSourceContext`](//platforms/android/source-context/#using-the-gradle-build-tool-plugin) option.
```

Reply:
I see @kahest suggested a fix to the URL here - is that the correct URL?
Or did you mean /platforms/android/source-context/#using-the-gradle-build-tool-plugin ?

If you meant to link to the Apple docs, I'd like to spend some more time rewriting this to make things more clear, as this section seems to only discuss Android and Gradle.

Reply:
https://github.com/getsentry/sentry-docs/pull/7144#discussion_r1229165720
I forgot to remove the `apple/guides` part. The first link points to a guide on how to upload native symbols and the second I wanted to be an explanation of `Source Context`, just in case users don't know what exactly it is.

Reply:
```suggestion
Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, DWARF debug files need to be uploaded for an Android app with [NDK](https://developer.android.com/ndk/) support.

To upload native symbols for React Native Android builds, use the Sentry Gradle Plugin's [`uploadNativeSymbols` option](/platforms/react-native/manual-setup/manual-setup/#enable-sentry-agp).

To enable [Source Context](/platforms/android/data-management/debug-files/source-context/) use the [`includeSourceContext`](/platforms/react-native/manual-setup/manual-setup/#enable-sentry-agp) option.
```